how-to/offline_cpu.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Description: Offline SOS CPUs except BSP before launch UOS
+
```

Comment:
Since this script requires root, I'd add a check for it:

```sh
[ $(id -u) -eq 0 ] || { echo >&2 "ERROR: run as root"; exit 1; }
```

Reply:
ok, done.